### ISSUE TITLE
crash opening coversation

### ISSUE THREAD

**Issue by mcginty** [WhisperSystems/TextSecure#4142]
```
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime: FATAL EXCEPTION: main
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime: Process: org.thoughtcrime.securesms.dev, PID: 30858
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime: java.lang.AssertionError: Selected type isn't present!
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.TransportOptions.getSelectedTransport(TransportOptions.java:65)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.TransportOptions.notifyTransportChangeListeners(TransportOptions.java:118)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.TransportOptions.setTransport(TransportOptions.java:113)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.TransportOptions.setDefaultTransport(TransportOptions.java:48)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.components.SendButton.setDefaultTransport(SendButton.java:79)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.ConversationActivity.handleSecurityChange(ConversationActivity.java:716)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.ConversationActivity.initializeSecurity(ConversationActivity.java:782)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.ConversationActivity.onCreate(ConversationActivity.java:218)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at org.thoughtcrime.securesms.PassphraseRequiredActionBarActivity.onCreate(PassphraseRequiredActionBarActivity.java:46)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.app.Activity.performCreate(Activity.java:5933)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1105)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2298)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2407)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.app.ActivityThread.access$800(ActivityThread.java:149)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1324)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.os.Handler.dispatchMessage(Handler.java:102)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.os.Looper.loop(Looper.java:211)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at android.app.ActivityThread.main(ActivityThread.java:5321)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at java.lang.reflect.Method.invoke(Native Method)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at java.lang.reflect.Method.invoke(Method.java:372)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1016)
09-28 15:42:10.577 30858-30858/org.thoughtcrime.securesms.dev E/AndroidRuntime:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:811)
```

**Comment by moxie0** [comment]
Duplicate of #4114

**Comment by mcginty** [comment]
crashing every time i open a group in 3.0.0 HEAD

**Comment by moxie0** [comment]
fixed in 3.0.0
